src/ui/src/components/command-palette/command-text-field.tsx

```diff
@@ -132,6 +132,11 @@ const useFieldStyles = makeStyles((theme: Theme) => createStyles({
     '&:not(:first-child)': {
       borderTop: theme.palette.border.unFocused,
     },
+    '& > $optionGroupHeading + ul > li': {
+      // Tells scrollIntoView to scroll a bit further up, so it can avoid being obscured by the sticky headers.
+      // Only applies when the header actually exists within that option's group.
+      scrollMarginTop: theme.spacing(3.125),
```

Comment:
This number seems a bit random to me, why 3.125?

Reply:
It matches the height of the headers defined above (computing the value would not be straightforward).